packages/init/test/cases/develop.default/develop.default.spec.js

```diff
@@ -93,8 +93,11 @@ describe("Initialize a new Greenwood project: ", function () {
     });
   });
 
-  after(function () {
+  after(async function () {
     runner.stopCommand();
-    runner.teardown([initOutputPath]);
+    // give the process a moment to release file handles (helps on Windows)
```

Comment:
I don't think we want this, this should be taken care of by #1573 which includes a fix to the test runner around `teardown` exit conditions